tools/grunt-tasks/minifyNames.js

```diff
@@ -1010,7 +1010,7 @@ function findPossibleFileConstants(parsedFile, theNames, ignoreConst, ignoreName
 }
 
 var dynamicIndex = 0;
-function createDynamicName(value) {
+function createDynamicName(value, isDupe) {
```

Comment:
This was generating the same "name" for the same string with a different case
eg. `traceId` and `traceID`